src/components/Layout.re

```diff
@@ -37,6 +39,27 @@ let make = () => {
             <img className=Styles.headerLogo src=smallIcon />
           </a>
           <ul className=Styles.navList>
+            <li className=Styles.navListItem>
+              <div className=Styles.navListItemToggle>
+                <span className=Styles.someMarginRight>
+                  <S>
+                    {usedtranslationModeContext->translationModeCrypto
+                       ? "EXPERT MODE" : "SIMPLE MODE"}
```

Comment:
Lets run another twitter poll that asks for different options on what these modes can be called. I quite like muggle mode, plain english, blockchain expert, etc...

Reply:
Okay cool, will deploy in the meantime with this though just so its not a blocker

Reply:
Yeah, good idea. Were you suggesting to have 3 modes?